modules/core/src/v2/coins/eth.ts

```diff
@@ -75,6 +99,7 @@ export class Eth extends BaseCoin {
   }
 
   static hopTransactionSalt = 'bitgoHopAddressRequestSalt';
```

Comment:
Please add `readonly` keyword to both of these statics.

Reply:
Fixed.